packages/mui-material/src/Button/Button.js

```diff
@@ -121,7 +121,10 @@ const ButtonRoot = styled(ButtonBase, {
           },
         }),
       ...(ownerState.variant === 'contained' && {
-        backgroundColor: (theme.vars || theme).palette.grey.A100,
+        backgroundColor:
+          (theme.vars || theme).palette.mode === 'dark'
```

Comment:
When using CSS variables, ideally we would use the same variable for both dark and light mode, and we would only switch the value of the variable. cc @siriwatknp should we add a variable for the inherit background of the button?

Reply:
@mnajdova @siriwatknp should we use this approach here? or is it good to go as it is?

Reply:
I would tend to use CSS variables, for e.g. as it is done for other components: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/experimental_extendTheme.js#L137. cc @siriwatknp